packages/main/src/TabContainer.ts

```diff
@@ -1019,16 +1019,18 @@ class TabContainer extends UI5Element {
 		}
 	}
 
-	_getRootTab(tab: Tab | undefined) {
-		while (tab?.hasAttribute("ui5-tab")) {
-			if (tab.parentElement!.hasAttribute("ui5-tabcontainer")) {
-				break;
-			}
+	_getRootTab(tab: Tab | undefined): Tab | undefined {
+		if (!tab?.parentElement) {
+			return undefined;
```

Comment:
Can you just change

`tab.parentElement!.hasAttribute("ui5-tabcontainer")`

to

`tab.parentElement?.hasAttribute("ui5-tabcontainer")`

from the original change?

Reply:
We prefer the loop and the function can be like:
```
_getRootTab(tab: Tab | undefined) {
		while (tab?.hasAttribute("ui5-tab")) {
			if (tab.parentElement?.hasAttribute("ui5-tabcontainer")) {
				break;
			}

			tab = tab.parentElement as Tab;
		}

		return tab;
	}
```

this won't change the return type.